src/modules/faucet/FaucetAssetsList.tsx

```diff
@@ -54,6 +58,20 @@ export default function FaucetAssetsList() {
     );
   }
 
+  if (currentMarketData.enabledFeatures?.external_faucet) {
```

Comment:
I think my suggestion earlier wasn't clear, but what I meant was to replace the button that is rendered on line 153 in this file with a button that will open a link to the external faucet. That way the faucet page looks the same for all testnets, and we are only changing the action that the faucet button does.